content/_code-samples/submit-and-verify/py/submit-and-verify.py

```diff
@@ -0,0 +1,40 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import AccountSet
+from xrpl.transaction import safe_sign_transaction, send_reliable_submission
+from xrpl.account import get_next_valid_seq_number
+from xrpl.ledger import get_latest_validated_ledger_sequence
+from xrpl.wallet import Wallet
+from xrpl.core import keypairs
+
+myAddr = "r31FLxNkfFJwRXbJw82RYB6Fa7jmfA3ix1"
+mySeed = "sn51LT5QeaPYhjM1HL9-----------"
+
+# Derive and initialize wallet
+public, private = keypairs.derive_keypair(mySeed)
+wallet_from_seed = Wallet(mySeed, 0)
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Construct AccountSet transaction
+tx = AccountSet(
+    account=myAddr,
+    fee="10",
+    sequence=get_next_valid_seq_number(address=myAddr, client=client),
```

Comment:
Use `safe_sign_and_autofill_transaction` So that ˋsequence` and `last_ledger_sequence` Get auto-filled :)